system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -5782,6 +5849,283 @@ class CLdapClient : public CInterface, implements ILdapClient
         else
             return -2;
     }
+
+    //Data View related interfaces
+
+    void createView(const char * viewName, const char * viewDescription)
+    {
+        if(viewName == nullptr || *viewName == '\0')
+            throw MakeStringException(-1, "Can't add view, viewname is empty");
+
+        if (stricmp(viewName, "Administrators") == 0 ||
+            stricmp(viewName, "Authenticated Users") == 0 ||
+            stricmp(viewName, "Directory Administrators") == 0)
+        {
+            throw MakeStringException(-1, "Can't add view, '%s' is a reserved name", viewName);
+        }
+
+        addGroup(viewName, nullptr, nullptr, m_ldapconfig->getViewBasedn());//TODO Save description
```

Comment:
At this time, we save the lfn/column info in the LDAP "description" field. We need to save it elsewhere (Cassandra?) but until we work that out there is nowhere to save the description